modules/light-clients/06-solomachine/misbehaviour_handle.go

```diff
@@ -4,6 +4,8 @@ import (
 	"github.com/cosmos/cosmos-sdk/codec"
 
 	commitmenttypes "github.com/cosmos/ibc-go/v6/modules/core/23-commitment/types"
+	sdk "github.com/cosmos/cosmos-sdk/types"
```

Comment:
Nit: our convention is to group the Cosmos SDK (or other Cosmos related) imports together, so that means that these two would go above.

Reply:
you should be able to run `make goimports` or `make goimports-all` which should correctly arrange the imports.

Reply:
Thank you for bringing this up, I have fixed it now.